src/main/java/org/terasology/logic/health/HealthCommands.java

```diff
@@ -15,6 +15,8 @@
  */
 package org.terasology.logic.health;
 
+import com.sun.xml.internal.ws.api.pipe.Engine;
+import org.mockito.internal.matchers.Null;
```

Comment:
minor style issue: not used import, using test dependencies in "production" code